### ISSUE TITLE
favorites not under search results when hidden on map

### ISSUE THREAD

**Issue by ristein** [osmandapp/Osmand#8700]
Title says it.
Favorites are not shown in search when not shown on map. This would be nice, since all those colored dots are not pretty on map. Also favorites should be found by that additional text like tag-words.

What version of OsmAnd are you using, on what device and Android/iOS version?
3.6.3 Android Oneplus 7pro custom rom
Do you use the offline maps offered within the OsmAnd app for download, or online (tile / raster) maps?
offline vector

**Comment by vshcherb** [comment]
If you disable layer selection in Configure Map, then favorites are searchable

**Comment by ristein** [comment]
you're right. i can disable all favorites under configure map and leave all favorites groups enabled to be able to search them. thats ok for me, thanks
